<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsForm</class>
 <widget class="QWidget" name="SettingsForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>5</y>
     <width>430</width>
     <height>350</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QTabWidget::pane
{ /* The tab widget frame */
	border-top: 2px solid #C2C7CB;
}


QTabWidget::tab-bar
{
	left: 5px; /* move to the right by 5px */
}


/* Style the tab using the tab sub-control. Note that it reads QTabBar _not_ QTabWidget */
QTabBar::tab
{
	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #E1E1E1, stop: 0.4 #DDDDDD, stop: 0.5 #D8D8D8, stop: 1.0 #D3D3D3);
	border: 2px solid #C4C4C3;
	border-bottom-color: #C2C7CB; /* same as the pane color */
	border-top-left-radius: 0px;
	border-top-right-radius: 0px;
	min-width: 8ex;
	padding: 2px;
}


QTabBar::tab:selected, QTabBar::tab:hover
{
	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #fafafa, stop: 0.4 #f4f4f4, stop: 0.5 #e7e7e7, stop: 1.0 #fafafa);
}


QTabBar::tab:selected
{
	border-color: #9B9B9B;
	border-bottom-color: #C2C7CB; /* same as pane color */
}


QTabBar::tab:!selected
{
	margin-top: 2px; /* make non-selected tabs look smaller */
}


</string>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::North</enum>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="documentMode">
    <bool>false</bool>
   </property>
   <property name="tabsClosable">
    <bool>false</bool>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="tabBarAutoHide">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="tabDrawing">
    <attribute name="title">
     <string>Drawing</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>88</y>
       <width>411</width>
       <height>146</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Pin up:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QColorSelector" name="pinUpColor">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_28">
        <property name="text">
         <string>Clock hand:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QColorSelector" name="clockHandColor">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Pin down:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QColorSelector" name="pinDownColor">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_27">
        <property name="text">
         <string>Dots:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QColorSelector" name="dotsColor">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Front:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QColorSelector" name="frontColor">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>Dot '12':</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QColorSelector" name="dot12Color">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Back:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QColorSelector" name="backColor">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLabel" name="label_29">
        <property name="text">
         <string>Sign '12':</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QColorSelector" name="sign12Color">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>Clock border</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QColorSelector" name="clockBorderColor">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QLabel" name="labelClockHandBorder">
        <property name="text">
         <string>Cl. hand border:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QColorSelector" name="clockHandBorderColor">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>411</width>
       <height>32</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QPushButton" name="pushButtonSaveColorsToFile">
        <property name="text">
         <string>Save to file</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="pushButtonSetDefaultColors">
        <property name="text">
         <string>Set to default</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="pushButtonLoadColorsFromFile">
        <property name="text">
         <string>Load from file</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>8</y>
       <width>253</width>
       <height>50</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="leftMargin">
       <number>1</number>
      </property>
      <item row="0" column="0">
       <widget class="QCheckBox" name="checkBoxDrawDots">
        <property name="text">
         <string>Draw dots</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_26">
        <property name="text">
         <string>Clock hand style:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="comboBoxClockHandStyle">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <item>
         <property name="text">
          <string>Rubik's style</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Simple line</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tabTimer">
    <attribute name="title">
     <string>Timer</string>
    </attribute>
    <widget class="QCheckBox" name="checkBoxSessionResetPopup">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>78</y>
       <width>191</width>
       <height>20</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>Popup before session reset</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>14</y>
       <width>96</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Timer updating:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxTimerUpdating">
     <property name="geometry">
      <rect>
       <x>165</x>
       <y>12</y>
       <width>107</width>
       <height>26</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <item>
      <property name="text">
       <string>On</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Off</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Seconds</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>46</y>
       <width>98</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Timer precision:</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton1_10">
     <property name="geometry">
      <rect>
       <x>166</x>
       <y>45</y>
       <width>42</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0.1</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton1_100">
     <property name="geometry">
      <rect>
       <x>225</x>
       <y>45</y>
       <width>50</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0.01</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton1_1000">
     <property name="geometry">
      <rect>
       <x>285</x>
       <y>45</y>
       <width>59</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0.001</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tabControls">
    <attribute name="title">
     <string>Controls</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>21</y>
       <width>401</width>
       <height>209</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="sizeConstraint">
       <enum>QLayout::SetMinimumSize</enum>
      </property>
      <item row="3" column="3">
       <widget class="QLineEditHotKey" name="key4m">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>6</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>3+</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QLineEditHotKey" name="key5m">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEditHotKey" name="key2p">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLabel" name="label_18">
        <property name="text">
         <string>4-</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLineEditHotKey" name="key2m">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QLineEditHotKey" name="key3m">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="4">
       <widget class="QLabel" name="label_24">
        <property name="text">
         <string>DL</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>5+</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="label_17">
        <property name="text">
         <string>3-</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEditHotKey" name="key4p">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEditHotKey" name="key5p">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>1-</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QLineEditHotKey" name="keyUL">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="4">
       <widget class="QLabel" name="label_25">
        <property name="text">
         <string>Mix</string>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QLabel" name="label_23">
        <property name="text">
         <string>DR</string>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QLineEditHotKey" name="keyUR">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="5">
       <widget class="QLineEditHotKey" name="keyDL">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>4+</string>
        </property>
       </widget>
      </item>
      <item row="4" column="5">
       <widget class="QLineEditHotKey" name="key_scramble">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLineEditHotKey" name="key1m">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEditHotKey" name="key1p">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string>2-</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QLabel" name="label_22">
        <property name="text">
         <string>UR</string>
        </property>
       </widget>
      </item>
      <item row="2" column="5">
       <widget class="QLineEditHotKey" name="keyDR">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="4">
       <widget class="QLabel" name="label_30">
        <property name="text">
         <string>x2</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>1+</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEditHotKey" name="key6">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>2+</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEditHotKey" name="key3p">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="QLineEditHotKey" name="key_y2">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QLabel" name="label_20">
        <property name="text">
         <string>y2</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QLabel" name="label_19">
        <property name="text">
         <string>5-</string>
        </property>
       </widget>
      </item>
      <item row="5" column="5">
       <widget class="QLineEditHotKey" name="key_x2">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="label_21">
        <property name="text">
         <string>UL</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_31">
        <property name="text">
         <string>z</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEditHotKey" name="key_z">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="6" column="3">
       <widget class="QLineEditHotKey" name="key_zp">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QLabel" name="label_32">
        <property name="text">
         <string>z'</string>
        </property>
       </widget>
      </item>
      <item row="6" column="5">
       <widget class="QLineEditHotKey" name="key_z2">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="6" column="4">
       <widget class="QLabel" name="label_33">
        <property name="text">
         <string>z2</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="labelKeyConflict">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>280</y>
       <width>211</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: red;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>401</width>
       <height>32</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_8">
      <item row="0" column="2">
       <widget class="QPushButton" name="pushButtonSaveControlsToFile">
        <property name="text">
         <string>Save to file</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="pushButtonLoadControlsFromFile">
        <property name="text">
         <string>Load from file</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="pushButtonSetDefaultControls">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Set to default</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QLineEditHotKey</class>
   <extends>QLineEdit</extends>
   <header location="global">qlineedithotkey.h</header>
  </customwidget>
  <customwidget>
   <class>QColorSelector</class>
   <extends>QFrame</extends>
   <header location="global">qcolorselector.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
